app/views/messages/index.html.erb

```diff
@@ -2,10 +2,10 @@
 
   <div class="min-w-full">
     <turbo-frame id="messages">
-      <%= render @conversation.messages %>
+      <%= render @messages %>
```

Comment:
This was rendering messages out of order, because the #sorted method wasn't being called on the collection. I was also querying for messages twice.